templates/volontulo/register.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block title %}Rejestracja w Volontulo{% endblock %}
+
+{% block content %}
+    <div class="col-xs-4">
```

Comment:
bootstrap `col-` elements must be in a `row` element
IMO it would be far smarter to add this wrapper in the other template where the `row` is present
this way we don't have to worry about incorrect column count in a row